lockable/provider_http.py

```diff
@@ -13,18 +16,43 @@ class ProviderHttp(Provider):
 
     def __init__(self, uri: str):
         """ ProviderHttp constructor """
+        self._configure_http_strategy(uri)
         super().__init__(uri)
 
+    def _configure_http_strategy(self, uri):
+        """ configure http Strategy """
+        retry_strategy = Retry(
```

Comment:
How long is total retry duration?

Reply:
calculation: `{backoff factor} * (2 ** ({number of total retries} - 1))`
with 5 retries and backoff_factor 0.5, retries happens `[0.0s, 0.5s, 1.0s, 2.0s, 4.0s].

I increase retries to 9 and `backoff_factor` to 1, so last retry happens after 2min8s  - does it sounds enough even with short service breaks ?